Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the terraform-k8s binary
+FROM golang:1.13 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
```

Comment:
I think the idea here is that as long as our deps don't change the build time will be improved since all the dependency downloads happen in a separate layer than the actual build of our binary.